VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -156,39 +183,72 @@ export class ParticipantsPanelComponent implements OnInit, AfterViewInit, OnDest
     }
 
     toggleMuteAll() {
-        this.videoCallService.muteAllParticipants(!this.isMuteAll);
+        this.logger.debug(`${this.loggerPrefix} Judge is attempting to toggle mute all status`, {
+            conference: this.conferenceId,
+            current: this.isMuteAll,
+            new: !this.isMuteAll
+        });
+        this.videoCallService.muteAllParticipants(!this.isMuteAll, this.conferenceId);
     }
 
     toggleSpotlightParticipant(participant: PanelModel) {
         const p = this.participants.find(x => x.id === participant.id);
-        this.videoCallService.spotlightParticipant(p.pexipId, !p.isSpotlighted);
+        this.logger.debug(`${this.loggerPrefix} Judge is attempting to toggle spotlight for participant`, {
+            conference: this.conferenceId,
+            participant: p.id,
+            pexipParticipant: p.pexipId,
+            current: p.isSpotlighted,
+            new: !p.isSpotlighted
+        });
+        this.videoCallService.spotlightParticipant(p.pexipId, !p.isSpotlighted, this.conferenceId, p.id);
```

Comment:
should log after too?

Reply:
a few here if going to change

Reply:
The callback (see line 105) will capture when this is successful.